www/include/configuration/configObject/host/listHost.php

```diff
@@ -94,9 +94,9 @@
 } else {
     //restoring saved values
     $search = $centreon->historySearch[$url]['searchH'] ?? null;
-    $poller = (int)$centreon->historySearch[$url]["poller"] ?? 0;
-    $hostgroup = (int)$centreon->historySearch[$url]["hostgroup"] ?? 0;
-    $template = (int)$centreon->historySearch[$url]["template"] ?? 0;
+    $poller = (int)($centreon->historySearch[$url]["poller"] ?? 0);
+    $hostgroup = (int)($centreon->historySearch[$url]["hostgroup"] ?? 0);
+    $template = (int)($centreon->historySearch[$url]["template"] ?? 0);
 }
```

Comment:
please can you add  : 
`$status = $centreon->historySearch[$url]["status"];` I removed it by mistake, when I was trying to debug the code to make the acceptance test :'(

Reply:
Done